src/components/YourCompany/SummaryDefineCompany.vue

```diff
@@ -39,17 +39,17 @@
     <v-divider />
 
     <div class="section-container">
-      <OfficeAddresses :inputAddresses="getOfficeAddresses" :isEditing="false" />
+      <OfficeAddresses />
```

Comment:
The OfficeAddresses component gets this itself - no need to pass in as prop.

Same for BusinessContactInfo on line 48 below.

Same for FolioInformation on line 52 below.

Reply:
Also, the sub-components manage their own editing state - no need to pass in.